## <span style = "color:#1A237E;">Hypothesis Two</span>
### <span style = "color:green;">Null Hypothesis</span>
Households in counties with higher poverty gaps are more likely to access credit.
### <span style = "color:green;">Alternative Hypothesis</span>
Households in counties with higher poverty gaps are less likely to access credit.
### <span style = "color:green;">Relevance</span>
Poverty gaps measure the depth of poverty, indicating how far the poor are from the poverty line. If households are far from the poverty line, they may face more difficulties accessing credit, thereby exacerbating their poverty situation

In [1]:
# Import required libraries
import pandas as pd
from scipy.stats import pearsonr

In [2]:
# Read data into pandas dataframe
data = pd.read_csv("overall_poverty_est.csv")
data.head()

residence_county  Headcount Rate (%)  Distribution of the Poor (%)  \
0          Baringo                39.6                           1.7   
1            Bomet                48.8                           2.7   
2          Bungoma                35.7                           3.4   
3            Busia                69.3                           3.6   
4  Elgeyo/Marakwet                43.4                           1.2   

   Poverty Gap (%)  Severity of Poverty (%) Population (ths)  \
0              9.7                      4.2              704   
1              9.3                      2.8              916   
2              9.5                      3.6            1,553   
3             22.3                      9.3              840   
4             13.4                      5.6              469   

   Number of Poor (ths)  Proportion of households that sought credit (%)  \
0                   278                                             44.4   
1                   447                                             19.5   
2                   555                                             32.8   
3                   583                                              5.5   
4                   204                                             43.1   

   Number of Households (ths)  \
0                         152   
1                         179   
2                         321   
3                         177   
4                          99   

   Proportion of households that sought and accessed credit (%)  \
0                                               98.6              
1                                               83.8              
2                                               58.0              
3                                               62.2              
4                                               98.7              

   Number of Households that sought credit (ths)  \
0                                             68   
1                                             35   
2                                            105   
3                                             10   
4                                             43   

   Distribution of the Poor (%).1  Poverty Gap (%).1  \
0                             2.0               10.8   
1                             2.1                5.6   
2                             3.5                9.5   
3                             3.4               17.5   
4                             1.4               10.8   

   Severity of Poverty (%).1 Population (ths).1  Number of Poor (ths).1  
0                        4.1                704                     291  
1                        1.6                916                     300  
2                        3.9              1,553                     503  
3                        7.2                840                     500  
4                        4.0                469                     210

In [3]:
# Display number of rows and columns
data.shape

(47, 16)

In [4]:
# Display statistical summary
data.describe()

Headcount Rate (%)  Distribution of the Poor (%)  Poverty Gap (%)  \
count           47.000000                     47.000000        47.000000   
mean            40.557447                      2.129787        12.085106   
std             16.291085                      1.109429         8.496751   
min             16.700000                      0.200000         2.400000   
25%             28.800000                      1.400000         7.350000   
50%             35.800000                      2.000000         9.400000   
75%             47.450000                      2.750000        13.400000   
max             79.400000                      5.200000        46.000000   

       Severity of Poverty (%)  Number of Poor (ths)  \
count                47.000000             47.000000   
mean                  5.306383            349.042553   
std                   5.254911            182.080125   
min                   0.500000             36.000000   
25%                   2.500000            231.000000   
50%                   3.500000            321.000000   
75%                   5.800000            455.500000   
max                  30.800000            860.000000   

       Proportion of households that sought credit (%)  \
count                                        47.000000   
mean                                         33.082979   
std                                          16.187170   
min                                           5.500000   
25%                                          21.300000   
50%                                          32.900000   
75%                                          43.100000   
max                                          66.100000   

       Number of Households (ths)  \
count                   47.000000   
mean                   242.808511   
std                    225.060343   
min                     30.000000   
25%                    127.000000   
50%                    210.000000   
75%                    277.500000   
max                   1503.000000   

       Proportion of households that sought and accessed credit (%)  \
count                                          47.000000              
mean                                           85.814894              
std                                            16.561138              
min                                            33.900000              
25%                                            84.050000              
50%                                            92.500000              
75%                                            97.550000              
max                                            99.200000              

       Number of Households that sought credit (ths)  \
count                                      47.000000   
mean                                       81.851064   
std                                        79.732972   
min                                         4.000000   
25%                                        39.500000   
50%                                        69.000000   
75%                                       108.500000   
max                                       510.000000   

       Distribution of the Poor (%).1  Poverty Gap (%).1  \
count                       47.000000          47.000000   
mean                         2.127660          10.314894   
std                          1.142404           6.053852   
min                          0.200000           3.000000   
25%                          1.300000           6.750000   
50%                          2.000000           9.100000   
75%                          2.650000          11.750000   
max                          4.900000          32.900000   

       Severity of Poverty (%).1  Number of Poor (ths).1  
count                  47.000000               47.000000  
mean                    4.463830              309.297872  
std                     3.569827              166.793795  
min                     0.800000      

In [5]:
# Display column headers
data.columns

Index(['residence_county', 'Headcount Rate (%)',
       'Distribution of the Poor (%)', 'Poverty Gap (%)',
       'Severity of Poverty (%)', 'Population (ths)', 'Number of Poor (ths)',
       'Proportion of households that sought credit (%)',
       'Number of Households (ths)',
       'Proportion of households that sought and accessed credit (%)',
       'Number of Households that sought credit (ths)',
       'Distribution of the Poor (%).1', 'Poverty Gap (%).1',
       'Severity of Poverty (%).1', 'Population (ths).1',
       'Number of Poor (ths).1'],
      dtype='object')

### <span style = "color:green;">Pearson correlation coefficient and p-value</span>

In [6]:
# Calculate Pearson correlation coefficient and p-value
corr, pval = pearsonr(data["Poverty Gap (%)"], data["Proportion of households that sought credit (%)"])

In [7]:
# Print results
print("Pearson correlation coefficient:", corr)
print("p-value:", pval)

Pearson correlation coefficient: -0.4777136954992022
p-value: 0.0006844717800419033


### <span style = "color:green;">Observations and Inferences</span>
1. The Pearson correlation coefficient between the `Poverty Gap (%)` and `Proportion of households that 
sought credit (%)` is `-0.4777`, which means that there is a moderate negative correlation between 
these two variables. This indicates that as the `Poverty Gap (%)` increases, the `Proportion of 
households that sought credit (%)` _decreases_, and vice versa.
2. The `p-value` is `0.000684`, which is less than the commonly used threshold of `0.05`. This suggests 
that the observed correlation between the two variables _is statistically significant_, and not 
likely to have occurred by chance.